lib/RateLimit/JsonRateLimitStore.php

```
@@ -0,0 +1,133 @@
+        return $this->data[$clientId] ?? [];
+    }
+
+    private function load(): void
+    {
+        $handle = fopen($this->filename, 'c+b');
+
+        if ($handle && flock($handle, \LOCK_SH)) {
+            $json = stream_get_contents($handle);
+
+            if ($json === false) {
+                error_log("Failed to read rate limit store from {$filename}");
```

PHPMD (2.15.0)

Avoid unused local variables such as '$filename'. (undefined)

(UnusedLocalVariable)